sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Create.java

```diff
@@ -586,9 +586,10 @@ public T apply(TimestampedValue<T> input) {
       throws CannotProvideCoderException {
     checkArgument(
         !Iterables.isEmpty(elems),
-        "Elements must be provided to construct the default Create Coder. To Create an empty "
-            + "PCollection, either call Create.empty(Coder), or call 'withCoder(Coder)' on the "
-            + "result PTransform");
+        "This Create class cannot deduce a Coder because it has zero " +
```

Comment:
@tgroh I don't think I'm following your comment.  There's no instance of Create here, it's a static method.  And the error message proposed in the patch says that the reason that it can't deduce a Coder is because it has zero elements.  Perhaps what you are saying is that the correct verb to use is "given" instead of "have".

I have doubts about your idea to say "Could not construct" because this function is lower level than the "of()" function -- is it true that this function would only get called during construction?  Let me take another crack at this and see what you think.